dace/sdfg/state.py

```diff
@@ -2841,17 +2889,25 @@ def add_state_after(self,
     ###################################################################
     # Traversal methods
 
-    def all_control_flow_regions(self, recursive=False, parent_first=True) -> Iterator['AbstractControlFlowRegion']:
+    def all_control_flow_regions(self, recursive=False, load_ext=False,
+                                 parent_first=True) -> Iterator['AbstractControlFlowRegion']:
         """ Iterate over this and all nested control flow regions. """
         if parent_first:
             yield self
         for block in self.nodes():
             if isinstance(block, SDFGState) and recursive:
                 for node in block.nodes():
                     if isinstance(node, nd.NestedSDFG):
-                        yield from node.sdfg.all_control_flow_regions(recursive=recursive, parent_first=parent_first)
+                        if node.sdfg:
+                            yield from node.sdfg.all_control_flow_regions(recursive=recursive, load_ext=load_ext,
+                                                                          parent_first=parent_first)
+                        elif load_ext:
+                            node.load_external(block)
```

Comment:
remnant from another PR?

Reply:
Must be due to a missed rebase, this is already in main: https://github.com/spcl/dace/blob/main/dace/sdfg/state.py#L2838